content/concepts/secure-comm/noise.md

```diff
@@ -6,5 +6,59 @@ aliases:
     - "/concepts/secure-comm/noise"
 ---
 
-<!-- ADD NOTICE -->
-Coming soon!
+## What is Noise?
+
+The [Noise Protocol Framework](https://noiseprotocol.org/) is a set of tools for creating
+secure communication protocols by combining cryptographic primitives into patterns with
+verifiable security properties. It provides a high level of flexibility, allowing users to
+choose from a range of cryptographic primitives and design patterns to suit their needs.
+However, it is up to the user to carefully consider the tradeoffs and make informed decisions
+to create a secure protocol.
+
+## Noise in libp2p
+
+noise-libp2p is an implementation of the Noise Protocol Framework used to establish a
+secure channel between two peers in the libp2p network. The protocol ID is
+`/noise`, and future versions of the specification may define new protocol IDs using the
+"/noise" prefix (e.g., `/noise/2`).
+
+When two peers connect, the transport upgrader negotiates
+which security and [multiplexing](../multiplex/overview.md) protocols to use using
+[multistream-select](https://github.com/multiformats/multistream-select).
+> A successor to multistream-select, called multiselect 2, is in development, but noise-libp2p is
+> compatible with the current upgrade process and multiselect 2.
```

Comment:
What is this?

Regardless, we don't need to mention multistream select here because the noise module takes an unencrypted data channel and returns an encrypted data channel. It doesn't know anything about multistream.